RpcNep5Tracker/Nep5Balance.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using System.Numerics;
+using Neo.IO;
+using Neo.Ledger;
+
+namespace Neo.Plugins
+{
+    public class Nep5Balance : StateBase, ICloneable<Nep5Balance>
+    {
+        public BigInteger Balance;
+        public uint LastUpdatedBlock;
```

Comment:
Is not serialized?

Reply:
Nice catch and fix in #67 
I added this field after the initial implementation and did not adjust the sizes. I should have written a unit test.